src/main/java/kitchenpos/application/TableService.java

```diff
@@ -1,73 +1,63 @@
-import kitchenpos.dao.OrderTableDao;
-import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.Order;
+import kitchenpos.dto.ChaneNumberOfGuestRequest;
+import kitchenpos.dto.ChangeEmptyRequest;
+import kitchenpos.dto.TableRequest;
+import kitchenpos.dto.TableResponse;
+import kitchenpos.port.OrderPort;
+import kitchenpos.port.OrderTablePort;
 import kitchenpos.domain.OrderTable;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.Arrays;
 import java.util.List;
-import java.util.Objects;
+import java.util.stream.Collectors;
 
 @Service
+@Transactional
 public class TableService {
-    private final OrderDao orderDao;
-    private final OrderTableDao orderTableDao;
+    private final OrderPort orderPort;
+    private final OrderTablePort orderTablePort;
 
-    public TableService(final OrderDao orderDao, final OrderTableDao orderTableDao) {
-        this.orderDao = orderDao;
-        this.orderTableDao = orderTableDao;
+    public TableService(final OrderPort orderPort, final OrderTablePort orderTablePort) {
+        this.orderPort = orderPort;
+        this.orderTablePort = orderTablePort;
     }
 
-    @Transactional
-    public OrderTable create(final OrderTable orderTable) {
-        orderTable.setTableGroupId(null);
+    public TableResponse create(final TableRequest request) {
+        final OrderTable orderTable = new OrderTable(request.getNumberOfGuests(), request.isEmpty());
+        orderTable.setTableGroupNull();
 
-        return orderTableDao.save(orderTable);
-    }
+        final OrderTable saveOrderTable = orderTablePort.save(orderTable);
 
-    public List<OrderTable> list() {
-        return orderTableDao.findAll();
+        return TableResponse.from(saveOrderTable);
     }
 
-    @Transactional
-    public OrderTable changeEmpty(final Long orderTableId, final OrderTable orderTable) {
-        final OrderTable savedOrderTable = orderTableDao.findById(orderTableId)
-                .orElseThrow(IllegalArgumentException::new);
+    @Transactional(readOnly = true)
+    public List<TableResponse> list() {
+        final List<OrderTable> orderTable = orderTablePort.findAll();
 
-        if (Objects.nonNull(savedOrderTable.getTableGroupId())) {
-            throw new IllegalArgumentException();
-        }
+        return orderTable.stream()
+                .map(TableResponse::from)
+                .collect(Collectors.toList());
+    }
 
-        if (orderDao.existsByOrderTableIdAndOrderStatusIn(
-                orderTableId, Arrays.asList(OrderStatus.COOKING.name(), OrderStatus.MEAL.name()))) {
-            throw new IllegalArgumentException();
-        }
+    public TableResponse changeEmpty(final Long orderTableId, final ChangeEmptyRequest request) {
+        final OrderTable savedOrderTable = orderTablePort.findById(orderTableId);
+        final List<Order> order = orderPort.findAllByOrderTableIdIn(Arrays.asList(orderTableId));
```

Comment:
```suggestion
        final Order order = orderPort.findById(orderTableId);
```
하나의 id로 조회하니 `List`보다는 `Optional`이나 `Order`를 응답받는게 낫지 않을까요? 🤔